packages/webdriverio/tests/utils/Timer.test.js

```diff
@@ -0,0 +1,57 @@
+import Timer from '../../src/utils/Timer'
+// import sinon from 'sinon'
```

Comment:
not needed, jest comes with a mocking interface

Reply:
Yeah forgot to remove this